articles/native-platforms/android/02-custom-login.md

```diff
@@ -1,10 +1,9 @@
 ---
-title: Auth0 Android Quickstarts - Custom Login
+title: Custom Login
 description: This tutorial will show you how to use the Auth0 authentication API in your Android project to create a custom login screen.
 seo_alias: android
 ---
 
-## Android - Custom Login Tutorial
 
 This is a simple quickstart that will show you how to add the Auth0 login capabilities while using a customized login screen.
 
```

Comment:
@tenmaster The Get Seed Project section is missing, please add it underneath the System Requirements.